src/components/Book.jsx

```diff
@@ -24,28 +44,40 @@ function Book({ book }) {
         </span>
         <div className="action-links-btns">
           <nav>
-            <Link className="action-link action-link-btns-item" to="/">
+            <button
+              type="button"
+              className="action-link action-link-btns-item"
+              to="/"
+            >
               Comments
-            </Link>
-            <Link className="action-link action-link-btns-item" to="/">
+            </button>
+            <button
+              type="button"
+              className="action-link action-link-btns-item"
+              onClick={handleRemove}
+            >
               Remove
-            </Link>
-            <Link className="action-link action-link-btns-item" to="/">
+            </button>
```

Comment:
- [ ] Ouch! this is really close but as part of the project requirements the button for removing the book should use the `<Button>` component you created instead of using the default HTML `<button>` element, so I'd suggest you update this accordingly.